ios/FluentUI/Pill Button Bar/PillButtonBar.swift

```diff
@@ -123,12 +115,13 @@ open class PillButtonBar: UIScrollView {
         }
     }
 
-    private var stackView: UIStackView = {
+    private var stackView = UIStackView()
```

Comment:
What's the reason for the change from stackView: UIStackView = {}() to stackView = UIStackView() with a different init function? Were the tokens not initialized yet, or something?

Reply:
Reopening for the same comments as on the `unreadDotLayer`, where we can just initialize this lazily and it _should_ work with the new token lookup.